compiler/pipes/register-kphp-configuration.cpp

```diff
@@ -19,7 +19,7 @@ bool RegisterKphpConfiguration::check_function(FunctionPtr function) const {
 void RegisterKphpConfiguration::on_start() {
   const auto klass = current_function->class_id;
   kphp_error(
-    !klass->members.has_any_instance_method() &&
+//    !klass->members.has_any_instance_method() &&
```

Comment:
?

Reply:
We now process `RegisterKphpConfiguration` before `ParseAndApplyPhpdocF`, and 
all this together is processed before filtering functions and the `KphpConfiguration` 
is left with a constructor due to which an error occurs.